src/reformat_libyuv.c

```diff
@@ -785,7 +785,7 @@ static void getLibYUVConstants(const avifImage * image, const struct YuvConstant
                 break;
             case AVIF_MATRIX_COEFFICIENTS_CHROMA_DERIVED_NCL:
                 switch (image->colorPrimaries) {
-                    case AVIF_COLOR_PRIMARIES_SRGB:
+                    case AVIF_COLOR_PRIMARIES_BT709:
```

Comment:
It is clearer to use `AVIF_COLOR_PRIMARIES_BT709` in these two places because of the `kYuvF709Constants` in line 791 and line 849.